### PR TITLE
stringext.1.3.1 - via opam-publish

### PR DESCRIPTION
**Pull request by rgrinberg**
Extra string functions for OCaml

Provides a single module named Stringext that provides a grab bug of often used
but missing string functions from the stdlib. E.g, split, full_split, cut,
rcut, etc.xtra string functions for OCaml

---
* Homepage: https://github.com/rgrinberg/stringext
* Source repo: https://github.com/rgrinberg/stringext.git
* Bug tracker: https://github.com/rgrinberg/stringext/issues

---
Pull-request generated by opam-publish v0.2.1

### CODE REVIEW

File: packages/stringext/stringext.1.3.1/descr [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Extra string functions for OCaml
+
+Provides a single module named Stringext that provides a grab bug of often used
+but missing string functions from the stdlib. E.g, split, full_split, cut,
+rcut, etc.xtra string functions for OCaml
```
Comment (yallop) [inline packages/stringext/stringext.1.3.1/descr]: Is there an editing error on the last line?  (also in 1.2.0 and 1.3.0)

Comment (rgrinberg) [inline packages/stringext/stringext.1.3.1/descr]: Yes. thanks for noticing. Do you mind if I send another PR to fix it after this one is merged. I'd rather just fix the old versions as well all in one go.

Comment (yallop) [inline packages/stringext/stringext.1.3.1/descr]: That's fine.  I'll merge this now.
